modules/Bio/EnsEMBL/Gene.pm

```diff
@@ -1527,6 +1507,42 @@ sub havana_gene {
   return $ott;
 }
 
+=head2 biotype
+
+  Arg [1]    : Arg [1] : (optional) String - the biotype to set
+  Example    : my $biotype = $gene->biotype;
+               my $biotype = $gene->biotype('protin_coding');
+  Description: Returns the Biotype object of this gene.
+               When no biotype exists, defaults to 'protein_coding'.
+  Returntype : Bio::EnsEMBL::Biotype
+  Exceptions : none
+  Caller     : general
+  Status     : Stable
```

Comment:
As discussed, status documentation is not meaningful as we have no policy to keep them up to date. Remove

Reply:
missed a couple of them, removed.